config/templates/contributorAddition.md

```diff
@@ -1,10 +1,9 @@
-Welcome to Zulip, @{commenter}! We just sent you an invite to collaborate on this repository at https://github.com/{repoOwner}/{repoName}/invitations. Please accept this invite in order to claim this issue and begin a fun, rewarding experience contributing to Zulip!
+Welcome to Zulip, @{commenter}! To get started, please take the following steps **in order** and mark them off when done:
```

Comment:
Arbitrary contributors won’t have permission to edit zulipbot’s comment, such as by checking its checkboxes.

Reply:
Ah, good to know!